pytorch/training/docker/1.12/py3/sdk2.6.0/Dockerfile.neuron.os_scan_allowlist.json

```diff
@@ -0,0 +1,766 @@
+{
```

Comment:
is this file copied from previous version or are the file contents updated with the latest scan?